spanner/session.go

```diff
@@ -849,12 +958,25 @@ func (p *sessionPool) sessionReady(s *session) {
 // or more requested sessions finished with an error. sessionCreationFailed will
 // decrease the number of sessions being created and notify any waiters that
 // the session creation failed.
-func (p *sessionPool) sessionCreationFailed(err error, numSessions int32) {
+func (p *sessionPool) sessionCreationFailed(err error, numSessions int32, isMultiplexed bool) {
 	p.mu.Lock()
 	defer p.mu.Unlock()
+	if isMultiplexed {
+		// Ignore the error if multiplexed session already present
+		if p.multiplexedSession != nil {
+			p.multiplexedSession.checkingHealth = false
+			p.multiplexedSessionCreationError = nil
+			p.mayGetMultiplexedSession <- true
```

Comment:
Is this needed? If there already was a multiplexed session, then there should be no waiters, meaning that we should also not need to send this signal, right?

Reply:
yes this is to unblock maintainer to continue check, if we don't send here it will keep on waiting and maintainer will go blocked